atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/ConnectionConfig.java

```diff
@@ -74,9 +74,15 @@ public int getMinConnections() {
 
     @Value.Default
     public int getMaxConnections() {
+        // todo (gmaretic): this needs to be bumped if we are reusing the connection pool
```

Comment:
This is not an actual todo, just a reminder that this config needs to be changed when we reuse -- will remove later